orchestra/admin.py

```diff
@@ -177,19 +177,19 @@ class TimeEntryAdmin(AjaxSelectAdmin):
 @admin.register(Todo)
```

Comment:
Testing checklist
- [ ] Add a nested todo list template to a role (e.g., our internal design checklist)
- [ ] check off some of the nested items
- [ ] refresh and ensure they got added and checked off properly
- [ ] open up the QA checklist (@Osmandina  can show you how), leave two comments, and copy/paste the results
- [ ] add a todo with a start date in the future. the task should appear in pending with that todo as the next step
 - [ ] add a todo with a due date in the future. task should appear in active with that todo as the next step